include/gazebo_mavlink_interface.h

```diff
@@ -87,10 +123,12 @@ class GazeboMavlinkInterface : public ModelPlugin {
     namespace_(kDefaultNamespace),
     motor_velocity_reference_pub_topic_(kDefaultMotorVelocityReferencePubTopic),
     imu_sub_topic_(kDefaultImuTopic),
+    imu_rate_(170),
```

Comment:
Why 170?

Reply:
According to @arwagoner tests, to get around 150hz rate at the FCU side, we would get it to be set at 170hz at Gazebo side. This can of course be removed or increased (we were trying to match a 150hz IMU rate).